### PR TITLE
Implement rising hub heights for wind; profile smoothing for offshore wind

### PR DESCRIPTION
**Pull request by nworbmot**
This PR makes several changes to improve the modelling of wind profiles:
- It raises the hub heights over the investment periods (from 80m to 100m for onshore wind, from 120m to 140m for offshore), to reflect larger wind turbines, and rising capacity factors.
- It implements atlite's [`windturbine_smooth`](https://github.com/PyPSA/atlite/blob/master/atlite/resource.py), as described by [Gorm Andersen et al's REatlas paper](https://arxiv.org/abs/1409.3353) and [Bloomfield et al 2022 Fig 5 from NationalGrid](https://essd.copernicus.org/articles/14/2749/2022/essd-14-2749-2022.pdf). This represents smoothing across weather cells, turbine availability and other factors deviating from raw wind power curves. This lowers capacity factors by around 20%.

The smoothing is not yet implemented for onshore wind, because we see from DTU's Global Wind Atlas that the ERA5 wind speeds are [a little too low over land in Germany](https://github.com/PyPSA/pypsa-eur/blob/811b4dc3aae1f71eb1f5d13559bd85611fbc3c52/scale_wind_data/gwa_era5.pdf). This is not an issue for offshore. Once the GWA correction is implemented, we can turn the smoothing on for onshore too.

FLH from [ISE Stromgestehungskosten 2024](https://www.ise.fraunhofer.de/de/veroeffentlichungen/studien/studie-stromgestehungskosten-erneuerbare-energien.html) for new turbines:

- onwind: 1800-2500-3200 (Binnen-Nord-Küste)                                                                                                                                                                                                                                                                                        
- offshore: 3200-3600-4500 (middle-far-best) 

Average CF for onshore in Germany in 2019 was [21.5%](https://www.statista.com/statistics/1468604/onshore-wind-power-capacity-factor-germany/) (1880 FLH).

Average capacity factors/FLH before the change in 2020:

- onwind: 20.2% / 1768
- offwind-ac: 47.4% / 4154
- offwind-dc: 58.8% / 5149

and in 2045:

- onwind: 23.3% / 2042
- offwind-ac: 47.9% / 4194
- offwind-dc: 58.6% / 5137

(changes due to changing distribution of wind inside Germany).

After the change in 2020:

- onwind: 22.2% / 1951
- offwind-ac: 38.7% / 3387
- offwind-dc: n/a

in 2045:

- onwind: 26.3% / 2305
- offwind-ac: 40.3% / 3531
- offwind-dc: 51.9% / 4547

Before asking for a review for this PR make sure to complete the following checklist:

- [x] Workflow with target rule `ariadne_all` completes without errors
- [x] The logic of `export_ariadne_variables` has been adapted to the changes
- [ ] One or several figures that validate the changes in the PR have been posted as a comment
- [ ] A brief description of the changes has been added to `Changelog.md`
- [x] The latest `main` has been merged into the PR
- [x] The config has a new prefix of the format `YYYYMMDDdescriptive_title`

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -105,12 +105,69 @@ atlite:
 renewable:
   onwind:
     cutout: europe-2019-sarah3-era5
+    resource:
+      smooth: true
```
Comment (lindnemi) [inline config/config.yaml:109]: looks like smoothing is turned on for onwind as well. is that intentional?

File: config/config.yaml [hunk 2]
```diff
@@ -105,12 +105,69 @@ atlite:
...
+          hub_height: 130.
+          P: 5.5
+        2040:
+          V: [3.0, 3.2, 3.5, 3.8, 4.0, 4.2, 4.5, 4.8, 5.0, 5.2, 5.5, 5.8, 6.0, 6.2, 6.5, 6.8, 7.0, 7.2, 7.5, 7.8, 8.0, 8.2, 8.5, 8.8, 9.0, 9.2, 9.5, 9.8, 10.0, 10.2, 10.5, 10.8, 11.0, 11.2, 11.5, 11.8, 12.0, 12.2, 12.5, 12.8, 13.0, 13.2, 13.5, 13.8, 14.0, 14.2, 14.5, 14.8, 15.0, 15.2, 15.5, 15.8, 16.0, 16.2, 16.5, 16.8, 17.0, 17.2, 17.5, 17.8, 18.0, 18.2, 18.5, 18.8, 19.0, 19.2, 19.5, 19.8, 20.0, 20.2, 20.5, 20.8, 21.0, 21.2, 21.5, 21.8, 22.0, 22.2, 22.5, 22.8, 23.0, 23.2, 23.5, 23.8, 24.0, 24.2, 24.5, 24.8, 25.0]
+          POW: [0.0, 0.127, 0.178, 0.237, 0.305, 0.381, 0.468, 0.564, 0.671, 0.789, 0.919, 1.061, 1.216, 1.385, 1.567, 1.765, 1.977, 2.205, 2.45, 2.711, 2.99, 3.287, 3.602, 3.937, 4.291, 4.665, 5.06, 5.477, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 5.5, 0.0]
+          hub_height: 140.
+          P: 5.5
   offwind-float:
```
Comment (lindnemi) [inline config/config.yaml:171]: offwind-float should probably also benefit from smoothing and hub height increases?

File: config/config.yaml (hunk 1 above)
Comment (nworbmot) [reply config/config.yaml:109]: I'm about to turn it off.

File: config/config.yaml (hunk 2 above)
Comment (nworbmot) [reply config/config.yaml:171]: It's not built in PyPSA-Ariadne, so we should probably just remove it. Also I'm not super familiar with the technology...

Comment (lindnemi) [reply config/config.yaml:171]: i saw some runs where it got built to some extent, but i'm happy to remove it

**Comment by lindnemi** [comment]
These changes might be a little too drastic. Offwind does not get built anymore (just when forced in).
Before:

Capacity-Electricity-Wind-Offshore



After:
Capacity-Electricity-Wind-Offshore


And looking at the European energy balance:

Before: 

image

After:
image



That's a 365H simulation

**Pull request merged**